File: python/liczby23/liczby23.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  liczby23.py

def liczby2():
    """
    Funkcja drukuje wszystkie liczby dwucyfrowe, w których cyfry nie powtarzają się. Funkcja zwraca ich liczbę. Wykluczone liczby: 11, 22, 33 itd.
    """
    suma = 0
    liczba = 0
    for i in range(1, 10):
        for j in range(0, 10):
            if i != j:
                print("{}{} ".format(i, j), end="")
                suma = suma + 1
                print(suma)
    return suma


def main(args):
    print("Liczb 2-cyfrowych:", liczby2())
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: C++/liczby23/liczby23.cxx
/*
 * liczby23.cxx
 */


#include <iostream>

using namespace std;

int liczby2(){
    int ile = 0;
    for(int i = 1; i<= 10; i++){
        for(int j = 0; j <= 10; i++){
            for(int k = 0; k < 10; k++){
            if(i != j && i != k && j != k){
                cout << i << j << k << " ";
                ile++;
            }
        }
        }
    }
    return ile;
}


int main(int argc, char **argv)
{
    int ile = liczby2();
	cout << "\n\nLiczb 2-cyfrowych:" << ile << endl;
	return 0;
}

